hlx_statics/blocks/inlinealert/inlinealert.css

```diff
@@ -146,4 +146,9 @@ main .inlinealert img {
     margin-top: 10px;
     width: 100%;
     height: 100%;
+}
+
+main .spectrum-InLineAlert--warning{
```

Comment:
We should use the specific orange color, rgb(246, 133, 17):
<img width="1466" alt="Screenshot 2024-12-17 at 7 54 14 AM" src="https://github.com/user-attachments/assets/76f3272e-7154-4df2-adaf-d1547227b409" />

Also, for completion, can we also add the other variants similar to InlineNestedAlert: https://github.com/AdobeDocs/adp-devsite/blob/main/hlx_statics/blocks/inline-nested-alert/inline-nested-alert.css#L12-L29

This next request is technically out of the scope of this PR, but it would also be a good opportunity to fix the colors for default (gray but should be blue), error (orange but should be red), and warning (gray but should be orange) in InLineNestedAlert as per the readme: https://github.com/adobe/aio-theme?tab=readme-ov-file#inlinealert-block-updated-2022-06-08

Would greatly appreciate these fixes. Otherwise, they will be easy to miss 🙏

Reply:
![image](https://github.com/user-attachments/assets/d298ef6d-10cd-4841-9c6a-f697bffd0f12)